code/src/reducers/todos.js

```diff
@@ -0,0 +1,62 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+
+
+const initialState = {
+    items: [
+        {id: 0, text:"call queen mother", isCompleted: false},
+        {id: 1, text:"make the app work", isCompleted: false},
+        {id: 2, text:"groceries and bolaget", isCompleted: false}
```

Comment:
You can probably delete these items since they're not displayed on the page

Reply:
Absolutely.